gnovm/tests/package_test.go

```diff
@@ -60,6 +60,9 @@ func runPackageTest(t *testing.T, dir string, path string) {
 	t.Helper()
 
 	memPkg := gno.ReadMemPackage(dir, path)
+	if memPkg.IsEmpty() {
+		panic(fmt.Sprintf("found an empty package %q", path))
```

Comment:
Why not simply `require.True` instead of panicking?

Reply:
Since I kind of change the behaviour of the `ReadMemPackage` method (to not panic when no `.gno` files are present) I wanted to keep the same behaviour whenever possible. This was to isolate the issue and minimize the need to check the impact of this change throughout the codebase. However, I completely agree with your point. Using require makes more sense in this context. I'll make that change. 👍